packages/backend/server/src/__tests__/copilot.spec.ts

```
@@ -1426,3 +1434,84 @@ test('should be able to manage context', async t => {
     }
   }
 });
+
+// ==================== workspace embedding ====================
+test.only('should be able to manage workspace embedding', async t => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)